src/extension.ts

```diff
@@ -1140,7 +1139,7 @@ class ExtensionManager implements vscode.Disposable {
   // depending on their 'ignoreCMakeListsMissing' state variable.
   enableWorkspaceFoldersFullFeatureSet() {
     for (const cmtFolder of this._folders) {
-      this.enableFullFeatureSet(!this.getFolderContext(cmtFolder.folder)?.ignoreCMakeListsMissing, cmtFolder.folder);
+      this.enableFullFeatureSet(!this.getFolderContext(cmtFolder.folder)?.ignoreCMakeListsMissing);
```

Comment:
Does it make sense to call this function N times if we don't store state per folder?

Reply:
We can call it once but only if we do a boolean processing folder by folder before that. I'll submit an update shortly.